### DISCUSSION TITLE
Protect and Serve talent stacks multiplicatively

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14055] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Security's Protect and Serve talent stacks multiplicatively if you have several security officers

It's a tier 1 talent so you can max out your reputation early in the campaign very easily:

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/e73cbebd-198b-4e23-8bc3-64be2266a472)

2 security officers:

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/28009c8e-c62c-46fe-98a2-e76bf91eeb3b)

### Reproduction steps

spawn 2 security officers
unlock protect and serve on each
pick an escort mission

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/ad671ddfd737e01a6ae63a8d66c97e1a9b80b839

**Comment by Regalis11** [comment]
Merged the fix to our private repo, coming up in a future update!
